sdk/keyvault/keyvault-keys/review/keyvault-keys.api.md

```diff
@@ -184,6 +184,13 @@ export type KeyCurveName = "P-256" | "P-384" | "P-521" | "P-256K";
 // @public
 export type KeyOperation = "encrypt" | "decrypt" | "sign" | "verify" | "wrapKey" | "unwrapKey" | "import";
 
+// @public
+export interface KeyOperationsOptions extends CryptographyOptions {
+    aad?: Uint8Array;
```

Comment:
Should be `additionalAuthenticatedData`.

Reply:
`additionalAuthenticatedData` is way better! Thank you!